shared/test/test_rds.rb

```diff
@@ -0,0 +1,268 @@
+require 'minitest/autorun'
+require_relative 'test_helper'
+require 'cdo/aws/rds'
+
+class TestRDS < Minitest::Test
+  def setup
+    @source_cluster_id = 'source-cluster'
+    @clone_cluster_id = 'cluster-clone'
+    @cluster_to_delete_id = 'delete-me-cluster'
+
+    @source_cluster = {
```

Comment:
Were these hashes manually composed, or auto-generated somehow? If manually composed, aside from the amount of time it must have taken to put these together (sunk cost), my concern moving forward is how we plan to maintain these fixtures over time as the rds API changes or evolves (as it inevitably will). Or if we wanted to go the auto-generated fixtures route, recording/replaying the tests through `vcr` might be another option.

Is it possible to trim down these fixtures so they only contain the relevant fields necessary to properly test the implementation, and leave out the rest of the noise? That might make this test easier to read/maintain.

Reply:
They were auto-generated.  I invoked the APIs from the Ruby console and copied/pasted the responses (changing unique identifiers to anonymous values).  I'll trim the unused attributes after implementing the parameter group cloning functionality.

Reply:
It was useful to have anonymized copies of real API responses, particularly when I added functionality to copy/delete ParameterGroups.  I've trimmed off most of the unused attributes on the stubbed data.